Mayhemfile

```diff
@@ -0,0 +1,6 @@
+project: mesh-processing-lib
```

Comment:
Have we tested the redirect ('>') wrapped in quotes with Mayhem? Does this do what you expect?

Reply:
Yes work as expected